examples/cockroachdb/cockroachdb-blueprint.yaml

```diff
@@ -0,0 +1,126 @@
+apiVersion: cr.kanister.io/v1alpha1
+kind: Blueprint
+metadata:
+  name: cockroachdb-blueprint
+actions:
+  backup:
+    secretNames:
+      - cockroachSecret
+    outputArtifacts:
+      cockroachDBCloudDump:
+        keyValue:
+          s3path: '/cockroachdb-backups/{{ .StatefulSet.Namespace }}/{{ index .Object.metadata.labels "app.kubernetes.io/instance" }}/{{ toDate "2006-01-02T15:04:05.999999999Z07:00" .Time  | date "2006-01-02T15-04-05" }}'
+    phases:
+      - func: KubeTask
+        name: backupToS3
+        args:
+          namespace: "{{ .StatefulSet.Namespace }}"
+          image: "cockroachdb/cockroach"
```

Comment:
Let's set the image tag of the latest release.

Reply:
Added